docs/mathematics/harshad-number.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Harshad Number
+slug: /mathematics/harshad-number
+---
+
+## Introduction
+
+A number is called a Harshad number if it is divisible by the sum of its digits. It can also be referred to as Niven number.
+
+## Explanation
+
+If the number under consideration is taken and the sum of its individual digits is calculated, if this resultant sum is divisible by the number itself then the number will be considered a Harshad number.
+
+Let's have one example: 
+```
+Input number : 120
+Output       : Is a Harshad Number
+
+Sequential steps :
+1 + 2 + 0 = 3
+120 % 3 = 0
+```
+
+## Algorithm
+
+- Loop through the individual digits of the number and add it to the `sum` variable.
+- Now check if the number is divisible by `sum`.
+- If it is then we can conclude that it is a Harshad number or else it is not.
+
+## Code
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+  defaultValue="py"
+  values={[
+    { label: 'Python', value: 'py', },
+    { label: 'Dart', value: 'dart', },
```

Comment:
Dart is not yet supported. Please use some other language. Prefer C++ or Java

Reply:
@HarshCasper harshad number is only there in python and dart...so is it fine to add support only for python or should I add code for it in java or c++ first and then make updates here